.github/workflows/test.yml

```diff
@@ -19,4 +19,11 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
+      - name: Run dpservice
+        run: docker run --rm --entrypoint ./dp_service.py --privileged -p1337:1337 --mount type=bind,source=/dev/hugepages,target=/dev/hugepages ghcr.io/ironcore-dev/dpservice-tester --no-init &
```

Comment:
It should use tester image with `main` label. 

> ghcr.io/ironcore-dev/dpservice-tester:main

I changed this also in metalnet.

Reply:
changed as requested